js/react/lib/components/contact-form/contact-form.component.tsx

```diff
@@ -0,0 +1,167 @@
+import { useState } from "react";
+import { Location } from '../../icons';
+import { CONTACT_FORM_ERROR_MESSAGES } from "./contact-form.const";
+import type { ContactFormProps } from "./contact-form.types";
+
+export default function ContactForm({ theme = "light" }: ContactFormProps) {
+  const [form, setForm] = useState({
+    name: "",
+    email: "",
+    location: "",
+    message: "",
+  });
+
+  const [errors, setErrors] = useState<Record<string, string>>({});
+  const [touched, setTouched] = useState<Record<string, boolean>>({});
+
+  const handleChange = (
+    e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    const { name, value } = e.target;
+    setForm(prev => ({ ...prev, [name]: value }));
+  };
+
+  const handleBlur = (e: React.FocusEvent<HTMLInputElement>) => {
+    const { name } = e.target;
+    setTouched(prev => ({ ...prev, [name]: true }));
+
+    if (!form[name as keyof typeof form]) {
+      setErrors(prev => ({
+        ...prev,
+        [name]: CONTACT_FORM_ERROR_MESSAGES.required,
+      }));
+    } else {
+      const updated = { ...errors };
+      delete updated[name];
+      setErrors(updated);
+    }
+  };
+
+  const handleSubmit = (e: React.FormEvent) => {
+    e.preventDefault();
+    const newErrors: Record<string, string> = {};
+    ["name", "email", "location"].forEach(field => {
+      if (!form[field as keyof typeof form]) {
+        newErrors[field] = CONTACT_FORM_ERROR_MESSAGES.required;
+      }
+    });
+
+    setErrors(newErrors);
+
+    if (Object.keys(newErrors).length === 0) {
+      console.log("Form submitted:", form);
+    }
+  };
+
+  const base = "w-full p-2 rounded-md border outline-none transition";
+  const styles = {
```

Comment:
Move static variant styles to `.const.ts` as in [/js/react/lib/components/button/button.const.ts](https://github.com/RustLangES/design-system-components/blob/main/js/react/lib/components/button/button.const.ts)